package.json

```diff
@@ -31,6 +32,11 @@
     "jsep": "^0.3.0"
   },
   "devDependencies": {
-    "budo": "^11.6.3"
-  }
+    "budo": "^11.6.3",
+    "webpack": "^4.41.6",
+    "webpack-cli": "^3.3.11"
+  },
+	"engines": {
+		"node": ">= 6.0.0"
```

Comment:
What is the engine requirement based on?

Reply:
Webpack had some mention of it being sometimes necessary to specify. I think it was webpack's minimum supported version of node.